examples/specs/text_tooltip_image.vl.json

```diff
@@ -1,18 +1,35 @@
 {
   "$schema": "https://vega.github.io/schema/vega-lite/v5.json",
-   "view": {"stroke": null},
+  "view": {
+    "stroke": null
+  },
   "description": "A simple chart with an image tooltip.",
   "data": {
     "values": [
-      {"image": "data/ffox.png"},
-      {"image": "data/gimp.png"},
-      {"image": "data/7zip.png"}
+      {
+        "image": "data/ffox.png"
+      },
+      {
+        "image": "../data/gimp.png"
```

Comment:
I don't think this is enough since other places depend on the previous location. Why did this break in the first place? How do we make sure that e.g. the editor still works correctly?